src/ui/components/Library/Team/utils.ts

```diff
@@ -12,7 +12,9 @@ export function parseQueryParams(query: ParsedUrlQuery) {
   let testRunId: string | undefined;
   let testId: string | undefined;
 
-  assert(!view || view === "runs" || view === "tests" || view === "recordings");
+  assert(
+    !view || view === "runs" || view === "tests" || view === "recordings" || view === "settings"
```

Comment:
Are these values enumerated somewhere?

Reply:
Not really. I found a `View` type in `src/ui/components/Library/Team/View/ViewContextRoot.tsx` but it doesn't include `settings` because it isn't used the same as the others.